.github/workflows/perf-models-impl.yaml

```diff
@@ -51,12 +66,28 @@ jobs:
       #    slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
       - name: Check perf report exists
         id: check-perf-report
-        if: ${{ !cancelled() }}
+        if: ${{ !cancelled() && (matrix.test-info.tracy && steps.download-profiler-artifact.outcome == 'success' || !matrix.test-info.tracy) }}
         run: |
-          ls -hal
-          export PERF_REPORT_FILENAME=Models_Perf_$(date +%Y_%m_%d).csv
-          ls -hal $PERF_REPORT_FILENAME
-          echo "perf_report_filename=$PERF_REPORT_FILENAME" >> "$GITHUB_OUTPUT"
+          TODAY=$(date +%Y_%m_%d)
+          PERF_REPORT_FILENAME_MODELS="Models_Perf_${TODAY}.csv"
+          PERF_REPORT_FILENAME_CCL="CCL_Perf_${TODAY}.csv"
+          if [ "${{ matrix.test-info.tracy }}" == "true" ]; then
```

Comment:
Yeah - this is the kind of complex logic that might be better served within a script or if it had its own pipeline.

We want to specifically run on the N300 single-card perf BMs?